doc/how-to-use.md

```diff
@@ -0,0 +1,400 @@
+## How to use multus-cni?
+
+### Prerequisite
+
+Kubelet must be configured to run with the CNI network plugin. Edit `/etc/kubernetes/kubelet` file and add `--network-plugin=cni` flags in `KUBELET\_OPTS `as shown below:
+
+```
+KUBELET_OPTS="...
+--network-plugin-dir=/etc/cni/net.d
+--network-plugin=cni
+"
+```
+
+### Install multus
+
+You could copy binary directory or could use daemonset yaml in multus repository.
+
+- copy multus binary
+
+Copy multus binary into CNI binary directory, usually `/opt/cni/bin`.
+
+```
+# Execute following commands at all Kubernetes nodes (i.e. master and minions)
+$ cp multus /opt/cni/bin
+```
+
+- use daemonset
+
+As [Quickstart](https://github.com/intel/multus-cni/README.md#quickstart-guide), you could apply as such:
+
+```
+# Execute following command at Kubernetes master
+$ cat ./images/{multus-daemonset.yml,flannel-daemonset.yml} | kubectl apply -f -
+```
+
+### Set up conf file in /etc/cni/net.d/
+
+You put CNI config file in `/etc/cni/net.d`. Kubernetes CNI runtime uses the alphabetically first file in the directory. (`"Note1"`, `"Note2"` are just comments, you can remove them at your configuration)
+
+```
+# Execute following commands at all Kubernetes nodes (i.e. master and minions)
+$ mkdir -p /etc/cni/net.d
+$ cat >/etc/cni/net.d/30-multus.conf <<EOF
+{
+  "name": "multus-cni-network",
+  "type": "multus",
+  "readinessindicatorfile": "/var/run/flannel/subnet.env",
+  "delegates": [
+    {
+      "Note1": "This is example, wrote your CNI config in delegates",
+      "Note2": "If you use flannel, you also need to run flannel daemonset before!",
+      "type": "flannel",
+      "name": "flannel.1",
+      "delegate": {
+        "isDefaultGateway": true
+      }
+    }
+  ],
+  "kubeconfig": "/etc/cni/net.d/multus.d/multus.kubeconfig"
+}
+EOF
+```
+
+For the detail, please take a look into [Configuration Reference](configuration.md)
+
+**You can use "clusterNetwork"/"defaultNetworks" instead of "delegates", see []() for the detail**
+
+As above config, you need to set `"kubeconfig"` in the config file for NetworkAttachmentDefinition(CRD).
+
+##### Which network will be used for "Pod IP"?
```

Comment:
When this file is rendered, the bold statement `**You can use "clusterNetwork"/"defaultNetworks" instead of "delegates", see []() for the detail**` and section `##### Which network will be used for "Pod IP"?` look like different sections and got lost for a second. Maybe add "NOTE:" in front the bold line, or turn both into some bulleted notes about the behaviour.

Reply:
Thank you for your comment! I agree indeed. Add "NOTE:".